tests/unit/CorrectNameRequest.spec.ts

```diff
@@ -3,7 +3,8 @@ import flushPromises from 'flush-promises'
 import { mount, Wrapper } from '@vue/test-utils'
 import CorrectNameRequest from '@/components/correct-name/CorrectNameRequest.vue'
 
-// mock the window.alert function to hide "[Vuetify] Unable to locate target XXX"
+// mock the window.alert function, console.warn to hide "[Vuetify] Unable to locate target XXX"
+console.warn = vitest.fn()
```

Comment:
1. Update comment.
2. Why are you doing this? There may be a valid reason but be aware that this hides console warnings that you might actually want to see when running the unit tests.

Reply:
<img width="1187" alt="Screenshot 2023-07-25 at 2 33 24 PM" src="https://github.com/bcgov/bcrs-shared-components/assets/54437031/672327eb-0da9-4119-ab01-4932dba5c41e">
To remove this

Reply:
Try to fix this by adding this at the top of the unit test file:
```
// Prevent the warning "[Vuetify] Unable to locate target #app"
document.body.setAttribute('id', 'app')
```

Reply:
I did that. But, when I used that, the following part didn't work
```
await this.showConfirmDialog(
            this.$refs.confirm,
            'Name Request Type Does Not Match Business Type',
            dialogContent
          )
```

Reply:
This is getting complicated. Just leave it for now I guess.